jaggeryapps/admin/serviceproviders/custom/controllers/custom/edit_finish.jag

```diff
@@ -513,11 +513,17 @@ function getAppAccessURL(storeProperties) {
 
 
 function getIDPInitiatedSSOURL(issuer) {
-    //TODO - Support tenants
-    //https://{Hostname}:{Port}/samlsso?spEntityID={SAML2 SSO Issuer name}
-    return application.get(MGT_TRANSPORT) + "://" + application.get(MGT_HOST_NAME) + ":" +application.get(HTTPS_PORT)
-    + "/samlsso?spEntityID=" +issuer;
 
+    var userInfo = session.get("userInfo");
+    var tenantDomain = userInfo.tenantDomain;
+
+    if (org.wso2.carbon.utils.multitenancy.MultitenantConstants.SUPER_TENANT_DOMAIN_NAME.equals(tenantDomain)) {
```

Comment:
Did the changes by considering proxyHost, proxyHTTPSPort and proxyContextPath in site.json